pt/lessons/advanced/concurrency.md

```diff
@@ -77,7 +81,7 @@ iex> spawn_link(Example, :explode, [])
 ** (EXIT from #PID<0.57.0>) evaluator process exited with reason: :kaboom
 ```
 
-Em determinados momentos não queremos que nosso processo vinculado falhe o atual. Para isso, precisamos interceptar as saídas. Quando saídas são interceptadas elas serão recebidas como uma mensagem de conjunto de variáveis: `{:EXIT, from_pid, reason}`. 
+Em determinados momentos não queremos que nosso processo vinculado falhe o atual. Para isso nós precisamos interceptar as saídas usando `Process.flag/2`. É usado a função do erlang [process_flag/2](http://erlang.org/doc/man/erlang.html#process_flag-2) para a `trap_exit` flag. Quando a interceptação sai (`trap_exit` é definida para `true`), sinais de saída são recebidas com uma tupla de mensagem: `{:EXIT, from_pid, reason}`.
```

Comment:
I believe this would sound better: "(...) interceptar as saídas usando `Process.flag/2`. **Ela usa a** função (...) para **a flag** `trap_exit`."

Also: "(...) sinais de saída **serão recebidos como** uma tupla (...)"

Reply:
@anderkonzen fixed.